webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -717,8 +717,8 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                     </thead>
                     <tbody>
                       {% for frame in parsed_dump.threads[crashing_thread].frames %}
-                        {% if frame.truncated %}
-                          <tr><td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td></tr>
```

Comment:
This changes the template to show the msg the processor created. That's good enough for now.